### PR TITLE
Bookmarked cv page

### PR DESCRIPTION
**Pull request by Mariya-Sharibchenko**

### CODE REVIEW

**Review by Karador — APPROVED** [review]

File: src/context/constants/pageCovers.ts [hunk 1]
```diff
@@ -7,3 +7,5 @@ export const ResponsesPageTitle = 'Отклики';
 export const InvitationsPageTitle = 'Приглашения';
 
 export const CVPageTitle = 'Профиль';
+
+export const BookmarkedCVPageTitle = 'Избранные кандидаты';
```
Comment (suicaed) [inline src/context/constants/pageCovers.ts:11]: Maybe it is better to wrap all these constants into one object?
And import it when it is needed (not spreaded properties), look:
```
<>{BookmarkedCVPageTitle</>
<>{SomeAnotherPageTitle}</>
```
and:
```
<>PageTitles.MyTitle</>
```

When you keep original object you have better context to understand where it comes from.
It is just my opinion, feel free to think about and to discuss :з

File: src/organisms/BookmarkedList/BookmarkedListContainer.tsx [hunk 2]
```diff
@@ -25,8 +25,10 @@ export const BookmarkedListContainer: React.FC<IBookmarkedListContainerProps> =
   }, []);
 
   const onSearchStudentClick = useCallback(() => {
-    searchInputValue !== '' && setStudentList(prevState => searchStudent(searchInputValue, prevState));
-  }, [searchInputValue]);
+   searchInputValue !== ''
```
Comment (suicaed) [inline src/organisms/BookmarkedList/BookmarkedListContainer.tsx:28]:
```
Boolean(Boolean(someValue) === Boolean(someValue !== ''))
```

I mean that you can use `Boolean()` to check if value is empty or not.

File: src/pages/Company/CompanyTemplate.tsx [hunk 3]
```diff
@@ -44,38 +43,57 @@ export const CompanyTemplate: React.FC<ICompanyTemplateProps> = ({
...
+              element={<BookmarkedCVPage students={StudentArray} CVInBookmarks={BookmarkedStudents} />}
+            />
+          </> :
           <>
             <Route
               path={CompanyMenuItems[0].pathTo}
               element={<HomePage students={StudentArray} studentsInBookmarks={BookmarkedStudents} />}
             />
+            <Route
+              path={CompanyMenuItems[2].pathTo}
```
Comment (suicaed) [inline src/pages/Company/CompanyTemplate.tsx:62]: `CompanyMenuItems[2]`
It looks strange.

**Review by suicaed — APPROVED** [review]
Well done!
I hope it will be useful for you to read it: [click](https://palantir.github.io/tslint/rules/no-magic-numbers/)

File: src/context/constants/pageCovers.ts (hunk 1 above)
Comment (Mariya-Sharibchenko) [reply src/context/constants/pageCovers.ts:11]: It's definitely better way, sometimes I just forget about it :)

**Pull request merged**